package/admin.js

```diff
@@ -28,5 +29,6 @@ Object.assign(Alchemy, {
   IngredientAnchorLink,
   Datepicker,
   Sitemap,
+  Tinymce: { ...Tinymce },
```

Comment:
Would this also work?

```suggestion
  Tinymce,
```

Reply:
Yes, it works